deepchem/models/tests/test_losses.py

```diff
@@ -381,3 +384,131 @@ def test_ShannonEntropy_pytorch(self):
             -np.mean([0.9 * np.log(0.9), 0.1 * np.log(0.1)])
         ]
         assert np.allclose(expected, result)
+
+    @pytest.mark.torch
+    def test_GlobalMutualInformation_pytorch(self):
+        """."""
+        from deepchem.feat.graph_data import BatchGraphData
+        from deepchem.models.torch_models.infograph import InfoGraphEncoder
+        from deepchem.models.torch_models.layers import MultilayerPerceptron
+        torch.manual_seed(123)
+
+        data, _ = self.get_regression_dataset()
+        batch = BatchGraphData(data.X).numpy_to_torch()
+
+        num_feat = 30
+        edge_dim = 11
+        dim = 4
+
+        encoder = InfoGraphEncoder(num_feat, edge_dim, dim)
+        encoding, feature_map = encoder(batch)
+
+        g_enc = MultilayerPerceptron(2 * dim, dim)(encoding)
+        g_enc2 = MultilayerPerceptron(2 * dim, dim)(encoding)
+
+        globalloss = losses.GlobalMutualInformationLoss()
+
+        excepted_global_loss = np.array(.0308)
+
+        global_loss = globalloss._create_pytorch_loss()(
+            g_enc, g_enc2).detach().numpy()
+
+        assert np.allclose(global_loss, excepted_global_loss, 1e-3)
+
+    @pytest.mark.torch
+    def test_LocalInformation_pytorch(self):
+        """."""
+        from deepchem.feat.graph_data import BatchGraphData
+        from deepchem.models.torch_models.infograph import InfoGraphEncoder
+        from deepchem.models.torch_models.layers import MultilayerPerceptron
+        torch.manual_seed(123)
+
+        data, _ = self.get_regression_dataset()
+        batch = BatchGraphData(data.X).numpy_to_torch()
+
+        num_feat = 30
+        edge_dim = 11
+        dim = 4
+
+        encoder = InfoGraphEncoder(num_feat, edge_dim, dim)
+        encoding, feature_map = encoder(batch)
+
+        g_enc = MultilayerPerceptron(2 * dim, dim)(encoding)
```

Comment:
This test can be simplified down by making a tensor with the appropriate shape. That way we don't have to tie this test to infograph.